_posts/adding-a-favicon.markdown

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title: An (overly) detailed explanation of how to submit a Pull Request
```

Comment:
I like the title but... (not to be too much of a stickler) could you capitalize it??

And I think you could just rename the markdown file to be like `2016-03-16-how-to-submit-a-pull-request.markdown`. I think that's fine (since it's more about that than adding a favicon).